src/Notifications.vue

```diff
@@ -213,6 +218,7 @@ const Component = {
   },
   methods: {
     destroyIfNecessary (item) {
+      this.$emit('onClick', item)
```

Comment:
```suggestion
      this.$emit('click', item)
```

Vue does not not seem to use "on" pattern